src/components/PastWorkOrders/MobileWorkingPastWorkOrdersView/__snapshots__/MobileWorkingPastWorkOrdersView.test.js.snap

```diff
@@ -0,0 +1,204 @@
+          class="govuk-warning-text__icon"
+        >
+          !
+        </span>
+        <div
+          class="govuk-warning-text__text"
+        >
+          <span
+            class="govuk-warning-text__assistive"
+          >
+            Warning
+          </span>
+          <p
+            class="govuk-!-margin-top-0 lbh-body-s lbh-!-font-weight-bold"
+          >
+            No work orders displayed
+          </p>
+          <p
+            class="lbh-body-xs govuk-!-margin-top-1"
+          >
+            Please contact your supervisor
+          </p>
+        </div>
+      </div>
+    </div>
+  </div>
+</DocumentFragment>
+`;
+
+exports[`MobileWorkingPastWorkOrdersView Component should match snapshot when there are work orders 1`] = `
+<DocumentFragment>
+  <div
+    class="mobile-work-order-container"
+  >
+    <div
+      class="date-picker-container"
+    >
+      <label
+        class="lbh-heading-h2"
+        for="date-picker"
+      >
+        Select date
+      </label>
+      <select
+        id="date-picker"
+        name="date-picker"
+      >
+        <option
+          data-testid="date-option"
+          value="1731801600000"
+        >
+          4-11-17
+        </option>
+        <option
+          data-testid="date-option"
+          value="1731715200000"
+        >
+          4-11-16
+        </option>
+        <option
+          data-testid="date-option"
+          value="1731628800000"
+        >
+          4-11-15
+        </option>
+        <option
+          data-testid="date-option"
+          value="1731542400000"
+        >
+          4-11-14
+        </option>
+        <option
+          data-testid="date-option"
+          value="1731456000000"
+        >
+          4-11-13
+        </option>
+        <option
+          data-testid="date-option"
+          value="1731369600000"
+        >
+          4-11-12
+        </option>
+        <option
+          data-testid="date-option"
+          value="1731283200000"
+        >
+          4-11-11
+        </option>
+      </select>
+    </div>
+    <svg
+      data-testid="spinner-undefined"
+      height="50"
+      stroke="#00703c"
+      viewBox="0 0 42 42"
+      width="50"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <g
+        fill="none"
+        fill-rule="evenodd"
+      >
+        <g
+          stroke-width="5"
+          transform="translate(3 3)"
+        >
+          <circle
+            cx="18"
+            cy="18"
+            r="18"
+            stroke-opacity=".5"
+          />
+          <path
+            d="M36 18c0-9.94-8.06-18-18-18"
+            transform="rotate(112.708 18 18)"
+          >
+            <animatetransform
+              attributeName="transform"
+              dur="1s"
+              from="0 18 18"
+              repeatCount="indefinite"
+              to="360 18 18"
+              type="rotate"
+            />
+          </path>
+        </g>
+      </g>
+    </svg>
+  </div>
+</DocumentFragment>
```

Comment:
You need to double check the contents of the snapshot tests. This contains a spinner